bin/compute_index.rb

```diff
@@ -2,10 +2,21 @@
 
 require_relative "../statistics/index"
 
+# Sort certain statistics numerically rather than alphabetically.
+def order(a, b)
+  # List of affected statistics (with numbers removed)
+  ["average_of_"].each do |id|
+    if(a[0].gsub(/\d/, '') == id and b[0].gsub(/\d/, '') == id)
+      return a[0].gsub(/\D/, '').to_i <=> b[0].gsub(/\D/, '').to_i
+    end
+  end
+  a <=> b
+end
+
 # Generate statistics index.
 puts "Computing statistics index."
 build_path = File.expand_path("../build", __dir__)
-list = STATISTICS.sort.map { |statistics_id, statistic_object| "- [#{statistic_object.title}](#{statistics_id})\n" }.join
+list = STATISTICS.sort{ |a, b| order(a, b) }.map { |statistics_id, statistic_object| "- [#{statistic_object.title}](#{statistics_id})\n" }.join
```

Comment:
We can generalize it for all statistics by padding numbers in titles for sorting:

```suggestion

list = STATISTICS
  .sort_by { |statistics_id, statistic_object| statistic_object.title.gsub(/\d+/) { |number| number.rjust(10, "0") } }
  .map { |statistics_id, statistic_object| "- [#{statistic_object.title}](#{statistics_id})\n" }
  .join

```